### PR TITLE
Add HTTPS to Steve's photo + update Kero's description

### PR DESCRIPTION
**Pull request by gwwatkin**
Was causing the website to appear as not secure:
Screenshot_20220124_233044

image

### CODE REVIEW

**Comment by gwwatkin** [comment]
Wondering if we should cache these images anyways, they are tiny after all

**Comment by alexnguyenn** [comment]
> Wondering if we should cache these images anyways, they are tiny after all

I am not too keen of having them on the repo, since they are not from us 🤔.

**Review by alexnguyenn — APPROVED** [review]

**Review by alexnguyenn — APPROVED** [review]

**Comment by Keelando** [comment]
> Was causing the website to appear as not secure: Screenshot_20220124_233044
> 
> image

Shows as secure on Chrome and Edge, but not Mozilla

File: src/pages/AboutUs.tsx [hunk 1]
```diff
@@ -144,9 +144,8 @@ const AboutUs = (): JSX.Element => {
                     sub="Assistant Professor at Simon Fraser University, Canada Research Chair"
                     avatarSrc="https://www.sfu.ca/content/sfu/physics/people/faculty/hoikwanl.img.1629284380.png"
                     description={
-                        "My research aims to solve the current practicality issues of quantum technology. " +
-                        "I will likely be facilitated by the freedom of Canada, friendliness of Canadians, " +
-                        "and fragrance of Tim Hortons coffee."
+                        "I aim at solving practicality issues of quantum technology, likely to be aided by the" +
```
Comment (Keelando) [inline src/pages/AboutUs.tsx:147]: "I aim _to solve_ practicality issues..."

Comment (gwwatkin) [reply src/pages/AboutUs.tsx:147]:
```suggestion
                        "I aim to solve practicality issues of quantum technology, likely to be aided by the" +
```

Comment (gwwatkin) [reply src/pages/AboutUs.tsx:147]: Though I think either is fine https://ludwig.guru/s/aim+at+solving

**Pull request merged**